### PR TITLE
Update quence.lua

### PR DESCRIPTION
**Pull request by millxing**
I added parameter options so that each track can be independently selected to output to midi, audio, midi & audio, crow JF or crow CV. I think it's working, but @justmat please test the crow options when you get a chance.

I wanted to add the ability to assign specific MIDI devices and channels to each track but I haven't thought of an elegant way to add this to the parameter menu without exploding the menu. With 4 tracks, 4 devices and 16 channels there are a lot of combinations. I'll give it some more thought, but please let me know if anyone has any ideas.

MIDI channels are still based on the track number. So if track 1 is MIDI and track 2 is Audio, track 3 will be channel 3 if MIDI is selected.

One other minor tweak -- I added two variables, pxcurve1 and pxcurve2 which control the randomization curves for notes and rests. Previously these were set at a constant. They are currently not changeable parameters, but can be easily tweaked in the code.

### CODE REVIEW

File: quence.lua [hunk 1]
```diff
@@ -128,23 +130,27 @@ function init()
     toniclist = {'C', 'C#', 'D', 'D#', 'E', 'F', 'F#', 'G', 'G#', 'A', 'A#', 'B'}
     tonicnum = 1
 
+    
     -- probabilities for randomness and rest frequencies
+    pxcurve1 = 1.5
```
Comment (tildebyte) [inline quence.lua:135]: 🌮  very much looking forward to playing with this

File: quence.lua [hunk 2]
```diff
@@ -82,32 +85,31 @@ local steps_copy = {}
 local rests_copy = {}
 local clk = beatclock.new()
 local reverse_play = {}
+local outnum = {}
+local oo = ' '
```
Comment (tildebyte) [inline quence.lua]: Could we have a more descriptive name here?

Comment (millxing) [reply quence.lua]: agreed. will fix.

**Comment by millxing** [comment]
dumb question: I tried adding "params:add_seperator()" to line 229, but keep getting an error. What am I missing?

**Comment by justmat** [comment]
My bad, it's a typo :D should be ``params:add_separator()``

**Comment by millxing** [comment]
i always have trouble with that word. Thanks!

**Pull request merged**